x-pack/plugins/fleet/server/services/epm/package_service.ts

```diff
@@ -49,7 +50,7 @@ export interface PackageClient {
   getRegistryPackage(
     packageName: string,
     packageVersion: string
-  ): Promise<{ packageInfo: RegistryPackage; paths: string[] }>;
+  ): Promise<{ packageInfo: ArchivePackage; paths: string[] }>;
```

Comment:
it seems weird to me that `getRegistryPackage` return an `ArchivePackage` should this be a new method, or should rename it `getPackage` ?

Reply:
calling it `getPackage` would be a good option. I've been in doubt about the naming because the whole flow is in `Registry` but it's not going to use the registry anymore.. I guess we'll have to refactor further